js/addToTask.js

```diff
@@ -0,0 +1,68 @@
+import {
+  CREATE_TASK_BUTTON,
+  CONTENT,
+  UL_LIST,
+  INPUT_TASK,
+  INPUT,
+  ADD_TASK,
+  TRASH_INPUT,
+  TODAY,
+  COMPLETE,
+  EDIT,
+  DELETE_TODO,
+  CLICK_EVENT,
+  HIDE,
+  AFTER_BEGIN,
+} from "/js/constants.js"; //named import
+
+export let taskList = [];
+export let id = 0;
+let dateString;
+function addToDo(taskName, id) {
+  const options = {
+    day: "numeric",
+    month: "numeric",
+    year: "numeric",
+  };
+  dateString = TODAY.toLocaleDateString("en-us", options);
+  const item = `<li class="item">                  
+                  <p class="text">${taskName}</p>
+                  <div class = "date" id = "${id}"> Created At:  ${dateString} </div>
+                  <img src= "icons/done.svg" data-job = "${COMPLETE}" id ="${id}"/>
+                  <img src = "icons/edit.svg" data-job = "${EDIT}" id = "${id}"/>
+                  <img src = "icons/delete.svg" data-job="${DELETE_TODO}" id="${id}"/>
+                  </li> `;
+
+  const position = AFTER_BEGIN;
+  UL_LIST.insertAdjacentHTML(position, item);
+}
+
+CREATE_TASK_BUTTON.addEventListener(CLICK_EVENT, function (event) {
+  if (INPUT_TASK.className === HIDE) {
+    INPUT_TASK.classList.remove(HIDE);
+  }
+  INPUT.focus();
+});
+
+ADD_TASK.addEventListener(CLICK_EVENT, function (event) {
+  const taskName = INPUT.value;
+  if (taskName) {
+    addToDo(taskName, id);
```

Comment:
Do we need validation for the input value?

Reply:
adding task only with spaces is handled with an error message in the updated code.